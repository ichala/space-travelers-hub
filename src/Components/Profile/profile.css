.profile-content {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: space-around;
}

.profile-title {
  margin: 10px;
}
