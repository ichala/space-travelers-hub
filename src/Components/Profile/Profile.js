import React from 'react';
import Missions from './Missions/Missions';
import './profile.css';
import DragonsProfile from './DragonsProfile/DragonsProfile';
import RocketProfile from './Rockets/Rocket';

function Profile() {
  return (
    <div className="profile-content">
      <div className="profile-missions">
        <h2 className="profile-title">My Missions</h2>
        <div className="mission-list">
          <Missions />
        </div>
      </div>
      <div className="profile-dragons">
        <h2 className="profile-title">My Dragons</h2>
        {' '}
        <DragonsProfile />
      </div>
      <div className="profile-rockets">
        <h2 className="profile-title">My Rockets</h2>
        <RocketProfile />
      </div>
    </div>
  );
}

export default Profile;
