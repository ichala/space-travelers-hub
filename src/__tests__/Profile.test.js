import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Profile from '../Components/Profile/Profile';
import store from '../Redux/configureStore';

describe('Test of Profile Components', () => {
  it('Profile Components Renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Profile />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
