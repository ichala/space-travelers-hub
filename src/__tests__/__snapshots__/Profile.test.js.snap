// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of Profile Components Profile Components Renders correctly 1`] = `
<div
  className="profile-content"
>
  <div
    className="profile-missions"
  >
    <h2
      className="profile-title"
    >
      My Missions
    </h2>
    <div
      className="mission-list"
    >
      <table
        className="zebra"
      >
        <thead>
          <tr>
            <th
              className="table-title"
            >
              Missions
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td
              className="table-title"
            >
              No missions available ..
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
  <div
    className="profile-dragons"
  >
    <h2
      className="profile-title"
    >
      My Dragons
    </h2>
     
    <table
      className="zebra"
    >
      <thead>
        <tr>
          <th
            className="table-title"
          >
            Dragons
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td
            className="table-title"
          >
            No Dragons Reserved...
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <div
    className="profile-rockets"
  >
    <h2
      className="profile-title"
    >
      My Rockets
    </h2>
    <table
      className="zebra"
    >
      <thead>
        <tr>
          <th
            className="table-title"
          >
            Rockets
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td
            className="table-title"
          >
            No missions available ..
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
